ReactWindows/ReactNative.Shared/UIManager/Events/InvokeEvent.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace ReactNative.UIManager.Events
+{
+    internal class InvokeEvent : Event
```

Comment:
>InvokeEvent [](start = 19, length = 11)

How did you decide to call this "Invoke" event rather than "AccessibilityTap" event?

Reply:
renamed to AccessibilityTapEvent

---
In reply to: [158562285](https://github.com/reseul/react-native-windows/pull/1#discussion_r158562285) [](ancestors = 158562285)